### ISSUE TITLE
Multiple SingleBar instances override each others options

### ISSUE THREAD

**Issue by BigBrainAFK** [AndiDittrich/Node.CLI-Progress#64]
When I instantiate multiple SingleBar isntances with different formats like

```js
const progBar = new cliProgress.SingleBar({
	format: 'Adding files: [{bar}] {percentage}% | ETA: {eta}s | {value}/{total} files'
}, cliProgress.Presets.shades_classic);
```
and

```js
const progBar2 = new cliProgress.SingleBar({
	format: 'Getting folders: [{bar}] {percentage}% | ETA: {eta}s | {value}/{total} folders'
}, cliProgress.Presets.shades_classic);
```

then `progBar` will also display the format chosen for `progBar2` so both act like having `format: 'Getting folders: [{bar}] {percentage}% | ETA: {eta}s | {value}/{total} folders'`.

Node: 13.10.1
cli-progress: 3.6.0

**Comment by AndiDittrich** [comment]
thanks, i'll check this

**Comment by BigBrainAFK** [comment]
Heres a test fiile if you want one.
[test.zip](https://github.com/AndiDittrich/Node.CLI-Progress/files/4410660/test.zip)

**Comment by AndiDittrich** [comment]
the issue is caused by the [options caching](https://github.com/AndiDittrich/Node.CLI-Progress/blob/master/lib/options.js#L2) - an artifact which has to be removed

**Comment by AndiDittrich** [comment]
fixed in v3.6.1
